README.md

```diff
@@ -134,9 +206,9 @@ DATASET_DATA_DIR/
 		
```

Comment:
I think moving the preparing datasets before the experiments is more clear, because for some experiments you would have already needed to do this right?

Reply:
"Preparing the datasets" section was moved before the experiments in the latest version, right?